src/vtbackend/Terminal.cpp

```diff
@@ -263,12 +263,12 @@ bool Terminal::processInputOnce()
 
     auto const readResult = readFromPty();
 
-    if (!readResult)
+    if (!readResult.has_value())
     {
+        terminalLog()("PTY read failed. {}", strerror(errno));
```

Comment:
that's a good thing to log, but maybe for everything except `EINTR` and `EAGAIN`. Maybe 2 lines below? Or was it intentionally placed above the if-condition?

Reply:
I used it for debug what is happening, i was not sure whether we were waiting for response or were infinitely looping somewhere else, so i would like to have it as a log ouput, maybe we can move it in another log tag ?